code/client/clrcore/External/BoneID.cs

```diff
@@ -27,27 +27,27 @@ public enum Bone
 		SKEL_Spine_Root = 57597,
 		SKEL_Spine0 = 23553,
 		SKEL_Spine1 = 24816,
-		SKEL_Spine2,
-		SKEL_Spine3,
+		SKEL_Spine2 = 24817,
+		SKEL_Spine3 = 24818,
 		SKEL_L_Clavicle = 64729,
 		SKEL_L_UpperArm = 45509,
 		SKEL_L_Forearm = 61163,
 		SKEL_L_Hand = 18905,
 		SKEL_L_Finger00 = 26610,
 		SKEL_L_Finger01 = 4089,
-		SKEL_L_Finger02,
+		SKEL_L_Finger02 = 64017,
 		SKEL_L_Finger10 = 26611,
 		SKEL_L_Finger11 = 4169,
-		SKEL_L_Finger12,
+		SKEL_L_Finger12 = 64097,
 		SKEL_L_Finger20 = 26612,
 		SKEL_L_Finger21 = 4185,
-		SKEL_L_Finger22,
+		SKEL_L_Finger22 = 64113,
 		SKEL_L_Finger30 = 26613,
 		SKEL_L_Finger31 = 4137,
-		SKEL_L_Finger32,
+		SKEL_L_Finger32 = 64065,
 		SKEL_L_Finger40 = 26614,
 		SKEL_L_Finger41 = 4153,
-		SKEL_L_Finger42,
+		SKEL_L_Finger42 = 64081,
```

Comment:
```suggestion
		SKEL_L_Finger42 = 4154,
```

The `64081` is for the right finger42 :)